src/Lykke.Common.Validation/Configuration/ConfigurationHelpers.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Linq;
+
+namespace Lykke.Common.Validation.Configuration
+{
+    /// <summary>Provides some static methods to simplify configuration</summary>
+    public static class ConfigurationHelpers
+    {
+        /// <summary>Splits unescaped string to chars</summary>
+        public static char[] UnescapedStringToCharsArray(this string config)
+        {
```

Comment:
Check the argument for null
If it possible, better to accept a more concrete type, otherwise IntelliSense will show this method for every string

Reply:
fixed